test_common/page_objects/et1/new_claim_page.rb

```diff
@@ -3,15 +3,62 @@ module EtFullSystem
   module Test
     module Et1
       class NewClaimPage < BasePage
+        include RSpec::Matchers
         set_url ::EtFullSystem::Test::Configuration.et1_url
-        section :main_content, '#content .main-section .main-content' do
-          element :start_a_claim_link, 'input[value="Start a claim"]'
+        section :feedback_notice, '.feedback-notice' do
+          element :language_picker, '.language-picker a'
+          element :language, :link_named, 'switch.language'
+          element :feedback_link, :paragraph_link_named, 'shared.feedback_link.feedback_statement_html'
         end
-
+        element :header, :h1_text, 'claims.new.header', exact: false
+        element :are_you_in_time_h2, :h2_text, 'claims.new.are_you_in_time.legend', exact: false
```

Comment:
Minor one - but the element names should be meaningful - no need to put things like 'h2' in them - as that is more the 'technical boring stuff' that your test suite probably doesnt care about - it just wants the 'are you in time' paragraph or something along those lines ?